lib/cartopy/util.py

```diff
@@ -56,26 +56,30 @@ def add_cyclic_point(data, coord=None, axis=-1):
     Adding a cyclic point to a data array, where the cyclic dimension is
     the right-most dimension
 
+    >>> from distutils.version import LooseVersion
     >>> import numpy as np
+    >>> if LooseVersion(np.__version__) >= '1.14.0':
```

Comment:
I'd be keen to hide this detail from the docs... http://www.sphinx-doc.org/en/stable/ext/doctest.html#directive-testsetup perhaps?

Reply:
Done.